src/main/scala/uk/gov/homeoffice/drt/AppEnvironment.scala

```diff
@@ -41,26 +43,47 @@ object AppEnvironment {
     env("AGGDB_SESSION_POOL_MAX").as[Int].default(10)
   ).parMapN(PostgreSQLConfig)
 
+  val httpClientConfig: ConfigValue[HttpClientConfig] = (
+    env("HTTP_CLIENT_CONNECTION_TIMEOUT").as[Int].default(2),
```

Comment:
I think it's useful to include the unit of time in the name for ENV VARS about time, `HTTP_CLIENT_CONNECTION_TIMEOUT_SECONDS` might be more obvious.